src/parser/wsdl.js

```diff
@@ -104,8 +106,18 @@ class WSDL {
         self.xmlnsInEnvelope = self._xmlnsMap();
         callback(err, self);
       });
+    }
 
-    });
+    if (self.syncLoad) {
+      loadUpSchemas()
+    } else {
+      process.nextTick(loadUpSchemas);
+    }
+  }
+
+  loadSync(callback) {
+    this.syncLoad = true
+    this.load(callback)
```

Comment:
I think we should reset `syncLoad` here. Otherwise, once `loadSync` is called, it will be always `synchronous`.

Reply:
It's not so intuitive in Node.js to have a synchronous method that takes a `callback`. A better way is to return the result directly, such as `const result = loadSync();`.

Reply:
I'll update to remove the callback